bam_consensus.c

```diff
@@ -1335,14 +1335,13 @@ int calculate_consensus_gap5(hts_pos_t pos, int flags, int depth,
 
 #ifdef K2
         int hb = 0;
-#define _ 0
-        static int X[16] = {_,0,1,_,2,_,_,_,3,_,_,_,_,_,_,_};
+        const int x = 0;
+        static int X[16] = {x,0,1,x,2,x,x,x,3,x,x,x,x,x,x,x};
         int i, base1 = X[p->base4];
         for (i = p->seq_offset-K2; i <= p->seq_offset+K2; i++) {
             int base = i >= 0 && i < p->b.core.l_qseq ? X[bam_seqi(seq,i)] : _;
```

Comment:
Missed one here, and similarly above. Would be nice if this could use `seq_nt16_int` instead of `X`, but those 4s might be a bit of a pain…